readme.md

```diff
@@ -56,6 +56,13 @@ Default: [`PATH`](https://github.com/sindresorhus/path-key)
 PATH to be appended.<br>
 Set it to an empty string to exclude the default PATH.
 
+##### execPath
+
+Type: `string`<br>
+Default: `process.execPath`
+
+Path to the current Node.js executable. Its directory is pushed to the front of PATH.
```

Comment:
`This can be either...` is missing here.

Reply:
Oops, sorry, thanks for fixing it!